custom_components/tahoma/cover.py

```diff
@@ -103,6 +107,16 @@ async def async_setup_entry(hass, entry, async_add_entities):
         SERVICE_MY, {}, "async_my",
     )
 
+    platform.async_register_entity_service(
```

Comment:
is there a way to only register the service if the command exists?

Reply:
Not that I am aware of. And even if this would be possible, you would still register it for all covers unfortunately. I figured out that you can pass a `SUPPORT_XXX` when you register the service, but this only works for predefined `SUPPORTED_FEATURES` I believe..

Reply:
fair enough

Reply:
You can loop on all the devices and check if at least one has the command.

Reply:
@tetienne I did implement some extra `SUPPORTED_FEATURES` for now. This looks for me like the cleanest 'core' way. (if they accept this .. ;-))